lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py

```diff
@@ -117,25 +159,26 @@ def process_body_from_alb_invocation(event):
 
     if not event_body_encoded:
         logger.info("event_body from alb record was not present")
-        logger.debug(event)
+        logger.info(event)
```

Comment:
we probably want to leave events and event bodies as debug logs

Reply:
please don't resolve convos without informing us as to why the requested change isn't being made. I still think this should be a debug event.